source/includes/mongodb-compatibility-table-swift.rst

```diff
@@ -4,6 +4,7 @@
    :class: compatibility-large
```

Comment:
This looks right for Swift. Our upcoming 1.3 release will add support for both MongoDB 5.1 and 5.2.